django/core/management/commands/shell.py

```diff
@@ -127,40 +120,66 @@ def python(self, options):
         # Start the interactive interpreter.
         code.interact(local=imported_objects)
 
-    def get_and_report_namespace(self, verbosity, no_imports=False):
-        if no_imports:
+    def get_auto_imports(self):
+        app_models_imports = [
+            f"{model.__module__}.{model.__name__}"
+            for model in reversed(apps.get_models())
+            if model.__module__
+        ]
+        # Future imports could include:
+        # "django",
+        # "django.conf.settings",
+        # "django.db.connection",
+        # "django.db.models",
+        # "django.db.reset_queries",
+        # "django.utils.timezone",
```

Comment:
Personally, I don't like implying future work in case the decision to do any of that changes (and we don't update the comment)
If this is aims to also document the format of the output, we could put that in a docstring
```suggestion
```

Reply:
I will push a proposal for a docstring.